examples/clipboard_.py

```diff
@@ -7,15 +7,16 @@
 .. tags:: gui
 """
 
+from qtpy.QtWidgets import QPushButton, QVBoxLayout, QWidget
 from skimage import data
-from qtpy.QtWidgets import QVBoxLayout, QPushButton, QWidget
+
 import napari
 
 # create the viewer with an image
 viewer = napari.view_image(data.moon())
 
 class Grabber(QWidget):
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
Doesn't `__init__` always returns None ? Is that the thing that Ruff always adds ?

Reply:
Yes and yes. 

uderground python `__new__` looks like 

```
def __new__(cls, *args, **kwargs):
    obj = super().__new__()
    cls.__init__(obj, *args, **kwargs)
    return obj
```

see returning anything from `__init__` is pointless.

Reply:
This requirement actually originates in [Mypy](https://mypy.readthedocs.io/en/stable/class_basics.html#annotating-init-methods): you have to annotate the `__init__` return type if it has no arguments (but can otherwise omit it).